Source/WebCore/platform/ios/PlaybackSessionInterfaceIOS.mm

```diff
@@ -119,6 +119,22 @@
 {
     return LogMedia;
 }
+
+uint32_t PlaybackSessionInterfaceIOS::ptrCount() const
+{
+    return CanMakeCheckedPtr<PlaybackSessionInterfaceIOS>::ptrCount();
```

Comment:
Ditto.

Reply:
Style checker complains when you inline a function in an exported class. Probably because it makes linkage weird? I didn't think too much about it. Just obeyed the bot.